src/_zkapauthorizer/tests/test_client_resource.py

```diff
@@ -192,6 +192,32 @@
 
 TRANSIENT_ERROR = u"something went wrong, who knows what"
 
+def get_dummyredeemer_public_key(plugin_name, node_config):
+    """
+    Get the issuer public key a ``DummyRedeemer`` has been configured with.
+
+    :param unicode plugin_name: The plugin name to use to choose a
+        configuration section.
+
+    :param _Config node_config: See ``from_configuration``.
+    """
+    section_name = u"storageclient.plugins.{}".format(plugin_name)
+    redeemer_kind = node_config.get_config(
+        section=section_name,
+        option=u"redeemer",
+    )
+    if redeemer_kind != "dummy":
+        raise ValueError(
+            "Cannot read dummy redeemer public key from configuration for {!r} redeemer.".format(
```

Comment:
I'd write a test for this but turns out there are no callers of this function in the follow-up PR that actually does something useful so I'll just delete the whole thing, instead.